src/middlewared/middlewared/plugins/apps/ix_apps/rollback.py

```diff
@@ -0,0 +1,31 @@
+import os
+import shutil
+
+from pkg_resources import parse_version
+
+from .path import get_installed_app_versions_dir_path
+from .utils import RE_VERSION
+
+
+def get_rollback_versions(app_name: str, current_version: str) -> list[str]:
+    rollback_versions = []
+    with os.scandir(get_installed_app_versions_dir_path(app_name)) as scan:
+        for entry in filter(
+            lambda e: e.name != current_version and e.is_dir() and RE_VERSION.findall(e.name), scan
```

Comment:
We should not have versions which are greater then current versions in the first place but probably better to account for the edge case if for whatever reason that happens to be so that we don't display those versions as valid?